src/Modules/ModParser.lua

```diff
@@ -75,6 +75,7 @@ local formList = {
 	["adds (%d+) to (%d+) (%a+) attack damage"] = "DMGATTACKS",
 	["adds (%d+)%-(%d+) (%a+) attack damage"] = "DMGATTACKS",
 	["(%d+) to (%d+) added attack (%a+) damage"] = "DMGATTACKS",
+	["adds (%d+) to (%d+) attack (%a+) damage to melee skills"] = "DMGATTACKSMELEE",
```

Comment:
We need another form for the reordering, but we shouldn't put tags in the forms at all.  It'd be much better to expand the functionality to support more than two suffixes, most likely.  If that's not worth the effort/causes too many other issues, these 2+ suffix outliers should just be in the `specialModList`

Reply:
Decided to use a special mod since parsing 3 suffixes leads to potentially undesirable side effects with other mods.